src/ucp/core/ucp_mm.c

```diff
@@ -886,7 +897,81 @@ ucp_mem_advise(ucp_context_h context, ucp_mem_h memh,
     return status;
 }
 
-static ucs_status_t
+static ucs_status_t ucp_remote_mkey_pack(ucp_context_h context, ucp_mem_h memh,
```

Comment:
1. why need separate function, and not just make it part of ucp_memh_pack?
seems only difference is the lock
2. i think this code should be in ucp_rkey.c, as "replacement" for ucp_rkey_pack